src/gd_kotlin.cpp

```diff
@@ -539,3 +544,11 @@ GDKotlin::GDKotlin() :
         is_gc_started(false),
         transfer_context(nullptr) {
 }
+
+void GDKotlin::register_members(jni::Env& p_env) {
+    auto map_entry = classes.front();
```

Comment:
I dont really like auto when no cast or explicit constructor call. Can we at least write ptr so that's explicit
```cpp
auto* map_entry{classes.front()};
```